flow-report/src/util.ts

```diff
@@ -48,6 +48,21 @@ export function getScreenshot(reportResult: LH.ReportResult) {
   return fullPageScreenshot || null;
 }
 
+export function convertChildAnchors(element: HTMLElement, index: number) {
```

Comment:
could we get an explanatory jsdoc on this one? converts them to what? in which context?

I haven't scrolled down yet, but I'm guessing it's converting anchors in the wrapped report, should it live over in the wrappers section?

Reply:
Yeah, it's pretty specific to the embedded report, so it can live there.